dashboard/app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,57 @@
+@import 'util';
+@import 'angularMaterial.shim';
+
+
+$dashboard-item-status-width: 120px;
+
+.dashboard-heading {
+  padding-left: 25px;
+}
+
+.dashboard-item {
+  position: relative;
+  width: 900px;
```

Comment:
@mDibyo do you know whether this will ultimately be responsive? Would be great if we could look at this from our mobile phones.